screens/HomeScreen.js

```diff
@@ -1,10 +1,13 @@
-import React from "react";
-import { View, Text } from "react-native";
+import React, { useEffect, useState } from "react";
```

Comment:
Did the changes in this file end up being part of this ticket? If not let's not include them unless there's a solid reason.